Add Windows GDI+ SKUs support note (#5023)
dotnet/dotnet-api-docs: includes/drawing.md

> [!WARNING]
> The `System.Drawing` namespace is not recommended for new development, due to not being supported within a Windows or ASP.NET service and it is not cross-platform. [ImageSharp](https://github.com/SixLabors/ImageSharp) and [SkiaSharp](https://github.com/mono/SkiaSharp) are recommended as alternatives.

> [!CAUTION]

> > On Windows `System.Drawing` depends on GDI+ native library, which is shipped as part of the OS. Some Windows SKUs like Windows Server Core or Windows Nano, don't include this native library as part of the OS, so exceptions will be thrown at runtime as we can't load this library.

Reviewer: ```suggestion
> On Windows, `System.Drawing` depends on the GDI+ native library, which is shipped as part of the OS. Some Windows SKUs, like Windows Server Core or Windows Nano, don't include this native library as part of the OS. Exceptions will be thrown at run time because the library can't be loaded.
```

Also, I think it's best to combine the Caution block and the Warning block into a single block with two bullets.

Author: And keep it just as !WARNING ?

Reviewer: Actually I think I would make it a caution block. From the contributor guide:

```
> [!CAUTION]
> Negative potential consequences of an action.

> [!WARNING]
> Dangerous certain consequences of an action.
```

Neither of the bullets seems "dangerous".

Author: Sounds good! Thanks.
